src/components/TransactionalInfo.tsx

```diff
@@ -70,6 +70,10 @@ const TransactionalInfo = ({ transaction: tx, addressHash, className, hideLeftAd
   let pendingToAddressComponent
   let lockTime: Date | undefined
 
+  const token = 'alph'
```

Comment:
Why moving it up again? Was there an issue with having it close to the render function? It serves no purpose to have it so high up, it's only used in the render function.

Reply:
My reason is in the future this will be replaced with proper code which I think will live up there - otherwise, there's no reason and is highly preferential

Reply:
sounds good 👍